.gitignore

```diff
@@ -1,4 +1,7 @@
+coverage/
 node_modules/
+packages/*/*.js
```

Comment:
And this one is weird too.

Reply:
Actually packages are built with `babel` which outputs the built file at the root of the package. I don't think we should commit these files should we? I can create a `.gitignore` file ignoring `/.*js` files if you think that's clearer.

Reply:
I'd add a comment ;-)

Reply:
Ok done in 50e4fd9f9af327d0c96afb73dcdf83114b160877